src/i18n/messages/en.messages.json

```diff
@@ -1,123 +1,124 @@
 {
-  "base": {
```

Comment:
This file shoudn't be changed at all. The number of spaces is changing but this file should remain the same, untouched in this PR.

Reply:
Ohh..I've fixed this